spec.json

```diff
@@ -0,0 +1,690 @@
+{
+  "Use": "corectl",
+  "Long": "Corectl contains various commands to interact with the Qlik Associative Engine. See respective command for more information",
+  "Flags": {
+    "traffic": {
+      "Name": "traffic",
+      "Shorthand": "t",
+      "Usage": "Log JSON websocket traffic to stdout",
+      "DefValue": "false"
+    },
+    "verbose": {
+      "Name": "verbose",
+      "Shorthand": "v",
+      "Usage": "Logs extra information",
+      "DefValue": "false"
+    }
+  },
+  "Commands": {
+    "build": {
+      "Use": "build",
+      "Short": "Reloads and saves the app after updating connections, dimensions, measures, objects and the script",
+      "Flags": {
+        "app": {
+          "Name": "app",
+          "Shorthand": "a",
+          "Usage": "App name, if no app is specified a session app is used instead."
+        },
+        "config": {
+          "Name": "config",
+          "Shorthand": "c",
+          "Usage": "path/to/config.yml where parameters can be set instead of on the command line"
+        },
+        "connections": {
+          "Name": "connections",
+          "Usage": "path/to/connections.yml that contains connections that are used in the reload. Note that when specifying connections in the config file they are specified inline, not as a file reference!"
+        },
+        "dimensions": {
+          "Name": "dimensions",
+          "Usage": "A list of generic dimension json paths"
+        },
+        "engine": {
+          "Name": "engine",
+          "Shorthand": "e",
+          "Usage": "URL to engine (default \"localhost:9076\")"
+        },
+        "headers": {
+          "Name": "headers",
+          "Usage": "Headers to use when connecting to qix engine",
+          "DefValue": "[]"
+        },
+        "measures": {
+          "Name": "measures",
+          "Usage": "A list of generic measures json paths"
+        },
+        "objects": {
+          "Name": "objects",
+          "Usage": "A list of generic object json paths"
+        },
+        "script": {
+          "Name": "script",
+          "Usage": "path/to/reload-script.qvs that contains a qlik reload script. If omitted the last specified reload script for the current app is reloaded"
+        },
+        "silent": {
+          "Name": "silent",
+          "Usage": "Do not log reload progress",
+          "DefValue": "false"
+        },
+        "ttl": {
+          "Name": "ttl",
+          "Usage": "Engine session time to live in seconds",
+          "DefValue": "30"
+        }
+      }
+    },
+    "catwalk": {
+      "Use": "catwalk",
+      "Short": "Opens the specified app in catwalk",
+      "Long": "Opens the specified app in catwalk. If no app is specified the catwalk hub will be opened.",
```

Comment:
I think we shouldn't include `Long` (or `Short`) in the spec since that may lead to a lot of changes to the spec when only updating documentation for a command.